### ISSUE TITLE
centering the page

### ISSUE THREAD

**Issue by FelixBenning** [MunifTanjim/minimo#293]
# Sidebar causes the page to stick to the left side

If found some interesting behavior when building the site without all the required configuration parameters:

![grafik](https://user-images.githubusercontent.com/29106652/94830574-00888e00-040c-11eb-8c97-2c5b4b2af7fa.png)

I narrowed it down to this setting

![grafik](https://user-images.githubusercontent.com/29106652/94830704-2150e380-040c-11eb-8f87-2d112182588c.png)

so apparently your sidebare causes the theme to hug the left wall?

![grafik](https://user-images.githubusercontent.com/29106652/94830830-47768380-040c-11eb-99bc-77df4d55589e.png)

# two menus

I am also not sure if I like the two menus

![grafik](https://user-images.githubusercontent.com/29106652/94832235-f10a4480-040d-11eb-99e1-d92a5e0594a2.png)

I just want to say that I am otherwise very impressed by the number of features. Awesome work! I'll have a look over the german translation to give back a little

**Comment by MunifTanjim** [comment]
> so apparently your sidebare causes the theme to hug the left wall?

Yeah. That was actually intentional. It might be possible to keep the site centered with sidebar enabled by tweaking some CSS. But I haven't tried to do that. 

> I am also not sure if I like the two menus

The demo site enables all the features to show what's possible. You can disable the top menu withby setting `params.settings.hideMainMenu` to `true`. Or hide the sidebar menu by removing `sidebar_menu` from `params.widgets.sidebar`

You can read this these posts to know about the ways you can enable/disable various features: https://minimo.netlify.app/tags/config/

**Comment by nicholasimon** [comment]
If you add a custom.css file in the static directory then the code below will centre it nicely (well for me it works). If the logo is too small then also add the logo css

      #sidebar {
	      margin-left: 10%
      }
      body {
	      margin-left: 8%	
      }
    .logo {
    	all: unset;
    }
